### ISSUE TITLE
77. video / logits -> pred probs -> pred labels

### ISSUE THREAD

**Issue by kutay-oksuzz** [mrdbourke/pytorch-deep-learning#262]
I don't understand here. When we did `loss_fn = nn.BCEWithLogitsLoss()` we didn't need to use the sigmoid function. How did we decide on the results when these data' are greater than 0.5 or less than 0.5?
----------------------------------------------------------------
# Find the predicted labels
y_preds = torch.round(y_pred_probs)

# In full (logits -> pred probes -> pred labels)
y_pred_labels = torch.round(torch.sigmoid(model_0(X_test.to(device))[:5]))

# Check for equality
print(torch.eq(y_preds.squeeze(), y_pred_labels.squeeze()))

# Get rid of extra dimension
y_preds.squeeze()
-----------------------------------------------
I couldn't understand the logic why we applied this part :(

**Comment by ishanluhani** [comment]
add your question in discussion tab not issue tab
